### ISSUE TITLE
Ligatures make OTF unreadable by Windows 10

### ISSUE THREAD

**Issue by ndaoust** [glyphr-studio/Glyphr-Studio-1#278]
When I export an OTF with any ligature, Windows 10 refuses its installation: "The requested file is not a valid font file." (If I remove the ligatures and try again, it works.)

**Comment by mattlag** [comment]
Could you send me the Glyphr Studio Project File, with the Ligature?  This is a new feature, so chances are, you found a bug!  Thanks!  You can either put it here or email it to mail@glyphrstudio.com

**Comment by ndaoust** [comment]
Here you go. Some of those ligatures are WIPs, but I also tried removing them all and using a single "oe" ligature, and the same bug occured.

Also, _something_ about the ligatures must have made its way to the OTF: I tried converting it to TTF, and while the resulting file was valid, all ligatures were replaced by a dimensionless blank while in-use. E.g. if I used the TTF in a ligature-recognizing application, "toenail" would appear as "tnail".

And for what it's worth, I was using the online version of Glyphr Studio.
[Nicode - Glyphr Project - 2018.9.15-4.24.44.txt](https://github.com/glyphr-studio/Glyphr-Studio-1/files/2389638/Nicode.-.Glyphr.Project.-.2018.9.15-4.24.44.txt)

Thank you.

**Comment by ndaoust** [comment]
Hmm, when I click on the text file link, my browser downloads the file, and it correctly loads in Glyphr Studio. Regardless, I will try again, isolated in my next comment.

**Comment by ndaoust** [comment]
[Nicode.-.Glyphr.Project.-.2018.9.15-4.24.44.txt](https://github.com/glyphr-studio/Glyphr-Studio-1/files/2389662/Nicode.-.Glyphr.Project.-.2018.9.15-4.24.44.txt)

**Comment by ndaoust** [comment]
[Nicode - Glyphr Project - 2018.9.15-4.24.44.txt](https://github.com/glyphr-studio/Glyphr-Studio-1/files/2389664/Nicode.-.Glyphr.Project.-.2018.9.15-4.24.44.txt)

**Comment by mattlag** [comment]
Oh, I got it :-) Blocked by FireFox

I will look into this, thank you!

**Comment by mattlag** [comment]
I got it - it looks like your font has some glyphs defined in the Private Use area (above 0xE000) which is totally valid... except Glyphr Studio also uses that area for Ligatures :-) so thank you for finding a bug, i'll get it fixed!

Additionally, it looks like those glyphs in your project are blank, which is why they are showing up as blank in the font file (and they are being double-defined, which is probably causing the font file error).  I'm also going to implement a "Delete Glyph" feature, which has been asked for but does not exist yet.  So, separate issue that will also be addressed.

![image](https://user-images.githubusercontent.com/5897669/45649489-4de16f00-ba80-11e8-9376-cfa7811354d8.png)

I'll update this issue when it's fixed. Thanks again!

**Comment by mattlag** [comment]
Fixed in 1.11.01

**Comment by ndaoust** [comment]
I procured 1.11.01 and was able to export and use my font (but will need to work on it some more...).

That was fast! I sent a modest contribution as thanks.

**Comment by mattlag** [comment]
Great to hear! Bugs I try to fix as soon as possible, especially for brand new features.  Strangely, when I was testing your project, the Window Font preview thing was showing up blank, but once it was installed it totally worked with all the Ligatures etc... It also showed up perfectly in Font Forge.

**Comment by ndaoust** [comment]
Same for me: showed up blank, worked fine regardless.
